tests/e2e/specs/drupal/preview.spec.ts

```diff
@@ -10,6 +10,8 @@ test.describe('instant preview', () => {
     await page
       .getByLabel('Title', { exact: true })
       .fill('Instant preview test');
+    await page.locator('#edit-submit').click();
+    await page.locator('li.tabs__tab a:text("Edit")').click();
```

Comment:
My only remark: Submitting the initial form should redirect straight to the regular edit form.